debian/patches/0018-focus-object-can-be-null.patch

```diff
@@ -0,0 +1,29 @@
+Index: maliit-framework-0.99.1+git20151118+62bd56b/input-context/minputcontext.cpp
+===================================================================
+--- maliit-framework-0.99.1+git20151118+62bd56b.orig/input-context/minputcontext.cpp	2020-03-26 18:35:00.000000000 -0500
++++ maliit-framework-0.99.1+git20151118+62bd56b/input-context/minputcontext.cpp	2020-03-26 19:00:22.927732927 -0500
+@@ -278,12 +278,14 @@
+ 
+ void MInputContext::setFocusObject(QObject *focused)
+ {
++    if (!focused) return;
```

Comment:
I'm not sure if we need a check here, because the value doesn't seem to be used. Does the crash happen in this function, in the `composeInputContext` the line after, or somewhere else?

Reply:
The value is used on the line after your reply, `composeInputContext->setFocusObject(focused)`

Reply:
But that wouldn't necessarily cause the pointer to be dereferenced, right.

Reply:
This would call into the platform plugin to set the focus object. It doesn't appear that ubuntu-keyboard provides this type of plugin, so I'll test with this removed.

Reply:
>But that wouldn't necessarily cause the pointer to be dereferenced, right.

That's the reason I asked whether the _crash_ happens inside it: I have no insight about what happens inside that line.

Reply:
In our case, our platform plugin doesn't supply a `composeInputContext`. However, in the spirit of smallest changes possible, I'll remove this change.